src/clj/harja/palvelin/palvelut/laadunseuranta/viestinta.clj

```diff
@@ -0,0 +1,80 @@
+(ns harja.palvelin.palvelut.laadunseuranta.viestinta
+  "Tässä namespacessa on palveluita laadunseuranta-asioihin liittyvään teksti-/sähköpostiviestintään."
+  (:require [taoensso.timbre :as log]
+            [clojure.core.match :refer [match]]
+            [harja.tyokalut.html :as html-tyokalut]
+            [harja.kyselyt.urakat :as urakat-q]
+            [harja.kyselyt.laatupoikkeamat :as laatupoikkeamat-q]
+            [harja.palvelin.integraatiot.sahkoposti :as sahkoposti]
+            [hiccup.core :refer [html]]
+            [harja.palvelin.komponentit.fim :as fim]
+            [harja.palvelin.palvelut.laadunseuranta.yhteiset :as yhteiset]
+            [harja.fmt :as fmt]
+            [harja.domain.tierekisteri :as tierekisteri]
+            [harja.pvm :as pvm])
+  (:use [slingshot.slingshot :only [try+ throw+]]))
+
+;; Viestien muodostus
+
+(defn laatupoikkeama-harja-url [urakka-id hallintayksikko-id]
+  (str "https://extranet.liikennevirasto.fi/harja#urakat/laadunseuranta/laatupoikkeamat?&hy="
```

Comment:
Tämä osoittaa nyt aina Harjan tuotantoympäristöön. Jos joku kirjaa LP:n testiympäristössä, niin URL on väärin. Onko tämä ongelma? Pohdin, että frontti lähettäisi LP:n mukana osoitteen samaiselle sivulle, jossa LP kirjattiin, mutta voi aiheuttaa ongelmia jos käyttäjä feikkaa URL:in. Mielellään siis muodostaisi URLin backendillä, mutta en ihan heti keksinyt miten nykyinen suoritusympäristö ja sen URL pääteltäisiin. Onko tähän ideoita vai mennäänkö tällä?